[caffe2] miniz fix -Wstrict-prototypes

johnhaitas opened the pull request:
Summary: this fixes -Wstrict-prototypes

Test Plan: eyes

Reviewed By: rmaz

Differential Revision: D44556017

albanD reviewed (approved):
SGTM!

albanD commented:
This change is already in the upstream miniz. So should be pretty safe for us to add as is.
If @johnhaitas has time to do a full upgrade to the latest version I'm happy to review it. But it might be a bit more work if anything was any BC-breaking change.

johnhaitas committed:
[caffe2] miniz fix -Wstrict-prototypes (#98027)

Summary:
this fixes -Wstrict-prototypes

Pull Request resolved: https://github.com/pytorch/pytorch/pull/98027

Test Plan: eyes

Differential Revision: D44556017

Pulled By: johnhaitas

fbshipit-source-id: bf60d29814ef852a37e0fba7e3de630a143bb4a5

johnhaitas commented:
@pytorchbot merge

albanD commented:
@pytorchbot merge
